oai-pmh mods representation

Natkeeran opened the pull request:
This PR provides a MODS representation.  This is done by powering a twig powered OAI-PMH Record view powered by a Drupal view consisting of a given Repository Item metadata.

Composer Changes
* places a MODS template (mods.html.twig) via post-update-cmd
* patches the module, which ran into this issue https://www.drupal.org/project/rest_oai_pmh/issues/3338647

Config Changes
* rest_oai_pmh.settings - module config, sets up MODS
* views.view.oai_pmh_item_data - provides the metadata that powers the twig 
* search_api.index.default_solr_index - some additional fields such as the ID needed for the above view
* views.view.oai_pmh_item_links - sets up the block needed to show the link
* context.context.display_oai_pmh_item_links - places the block

Additional Notes
* Some of the other changes to the composer are due to an issue in building isle-dc.
* When sync from isle-dc, some configs are being updated with ip or domain, I am not sure how to avoid this!

rosiel commented on config/sync/media.settings.yml:
```diff
@@ -3,4 +3,4 @@ _core:
 iframe_domain: ''
 oembed_providers_url: 'https://oembed.com/providers.json'
-standalone_url: false
+standalone_url: true
```
unset this - it's being set by Isle-DC using code in buildkit. https://github.com/Islandora-Devops/isle-dc/issues/326

Natkeeran commented:
@rosiel 

I've changed the context as suggested, and also cleaned up the configs that gets into the PR for this feature.  It was easier to create another PR: https://github.com/Islandora-Devops/islandora-starter-site/pull/61.
